### ISSUE TITLE
BatchMarketStocksRequestBuilder doesn't allow to add a range parameter for chart type

### ISSUE THREAD

**Issue by jhassony** [WojciechZankowski/iextrading4j#31]
With `ChartRequestBuilder ` I'm able to use `withChartRange(ChartRange range)`.

I would expect to be able to use the same parameter with `BatchMarketStocksRequestBuilder `as well, since it is supported by the batch market end point of IEX (https://iextrading.com/developer/docs/#batch-requests).
As the documentation states:

> 
> **range** 
> 
> - Optional 
> - Used to specify a chart range if chart is used in types parameter.

I'm using the latest version 2.2.4.

I think usage should be the same, like the following:
```
final IEXTradingClient iexTradingClient = IEXTradingClient.create();
final BatchMarketStocksRequestBuilder batchRequest = new BatchMarketStocksRequestBuilder()
				.addType(BatchStocksType.CHART)
                                .withChartRange(ChartRange.THREE_MONTHS);
		
		for (String symbol : symbols) {
			batchRequest.withSymbol(symbol);
		}
		
		Map<String, BatchStocks> batchResult = iexTradingClient.executeRequest(batchRequest
				.build());
```

**Comment by WojciechZankowski** [comment]
Hey,

Thank you for pointing it out. I added method .withChartRange to the builder. Also I put there a method .withSymbols so you dont need to iterate it like in your example. Take a look:

```java
final IEXTradingClient iexTradingClient = IEXTradingClient.create();
final BatchMarketStocksRequestBuilder batchRequest = new BatchMarketStocksRequestBuilder()
				.addType(BatchStocksType.CHART)
                                .withChartRange(ChartRange.THREE_MONTHS);
		                .withSymbols(symbols)
Map<String, BatchStocks> batchResult = iexTradingClient.executeRequest(batchRequest
				.build());
```

It is available in version 2.2.5

```
<dependency>
	<groupId>pl.zankowski</groupId>
	<artifactId>iextrading4j-all</artifactId>
	<version>2.2.5</version>
</dependency>
```

Let me know if it works for you.

Thanks,

Wojtek

**Comment by jhassony** [comment]
Thanks! it definitely works!
